src/main/java/com/connexta/store/adaptors/StorageAdaptorRetrieveResponse.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.adaptors;
+
+import java.io.InputStream;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import org.springframework.http.MediaType;
+
+@Data
+public class StorageAdaptorRetrieveResponse {
```

Comment:
Turn the FileRetrieveResponse class into interface. Have the StorageAdaptorRetrieveResponse implement the interface.